docker-java-transport-httpclient5/src/main/java/com/github/dockerjava/httpclient5/ApacheDockerHttpClient.java

```diff
@@ -23,13 +25,18 @@ public Builder sslConfig(SSLConfig value) {
             return this;
         }
 
+        public Builder connectionPool(ConnectionPoolConfig conf) {
```

Comment:
Since in most cases the route is always the same, I recommend we simplify this to just `Integer connectionPoolSize`

Reply:
That would work for our use case so... sure. I just thought it might be more logical to just copy the underlying config options exactly. It is also slightly more future proof. Don't really feel strongly about it either way so I'll do as you ask.

Reply:
Done!